buildtest/cli/report.py

```diff
@@ -59,6 +76,7 @@ class Report:
         "state": {"description": "Filter by test state", "type": "PASS/FAIL"},
         "tags": {"description": "Filter tests by tag name", "type": "STRING"},
     }
+    color_options = list(ANSI_COLOR_NAMES.keys())
```

Comment:
why is this line in there? this must be a typo

Reply:
removed